openstack/networking/v1/subnets/requests.go

```diff
@@ -31,16 +30,16 @@ type ListOpts struct {
 	GatewayIP string `json:"gateway_ip"`
 
 	// Specifies the IP address of DNS server 1 on the subnet.
-	PRIMARY_DNS string `json:"primary_dns"`
+	PrimaryDns string `json:"primary_dns"`
 
 	// Specifies the IP address of DNS server 2 on the subnet.
-	SECONDARY_DNS string `json:"secondary_dns"`
+	SecondaryDns string `json:"secondary_dns"`
 
 	// Identifies the availability zone (AZ) to which the subnet belongs.
 	AvailabilityZone string `json:"availability_zone"`
 
 	// Specifies the ID of the VPC to which the subnet belongs.
-	VPC_ID string `json:"vpc_id"`
+	VpcID string `json:"vpc_id"`
```

Comment:
We should leave acronyms if possible, same we do for `VpcID`: `PrimaryDNS`, `SecondaryDNS`